package.json

```diff
@@ -40,6 +40,7 @@
     "build": "lerna run build",
     "check-license": "./scripts/check-license.sh",
     "coverage": "lerna run coverage",
+    "depcheck": "(node scripts/generateDepcheckrc.js && depcheck packages/jaeger-ui && rm -f packages/jaeger-ui/.depcheckrc.json;) || (EXIT_CODE=$?; rm -f packages/jaeger-ui/.depcheckrc.json; exit $EXIT_CODE)",
```

Comment:
This would store the exit code (255) in case there would be some unused dependency, then would delete the file, and finally exits the CI, stopping further jobs.

Reply:
if depcheck fails, the `rm` command will not be run because of `&&`

Reply:
can depcheck be invoked programmatically as a module? Then you could call it from the custom script you added

Reply:
> if depcheck fails, the rm command will not be run because of &&

I have included an `||` condition in case it fails, the other remove command would remove it

Reply:
> can depcheck be invoked programmatically as a module

Yes, it can be invoked. In that case, we wouldn't have to handle write or delete ops.